blogposts/2021/show-us-your-calendar.md

```diff
@@ -0,0 +1,166 @@
+
+Vanesa is careful to note, however, that anyone considering changing their schedule needs to do a cost-benefit analysis. Vanesa works 32 hours across four days instead of 40 across 5, meaning she has to consider the consequences of working part time.
+
+As much as she loves her schedule, she recommends taking account of the costs: “Consider the fact that, realistically, it will slow down your career path a little bit.” Sacrificing one day a week may be the best choice for your life, but being ⅕ less available than your coworkers will affect your career “one way or another.”
+
+## TJ DeVries, software engineer, streams on Fridays
+
+[TJ DeVries](https://about.sourcegraph.com/handbook/company/team#tj-devries), software engineer at Sourcegraph and core maintainer for open source Vim-based text editor [Neovim](https://neovim.io/), streams almost all day every Friday. How? He works four tens: four days a week, ten hours each.
+
+> “I could work in the middle of the night for two nights a week and if I ship all the same amount of stuff, I don’t think people would care. - TJ DeVries, software engineer
+
+TJ acknowledges it’s not his _ideal_ schedule. “My preferred hours would be working in the middle of the night but that doesn’t work with a family,” he says, laughing. TJ has an 11-month-old baby boy, a wife, and responsibilities at church: all of which make weeknights busy. Working four tens allows him to handle those responsibilities at night and have Fridays free for streaming his [live coding sessions](https://www.twitch.tv/teej_dv).
+
+![TJ coding live on Twitch](https://storage.googleapis.com/sourcegraph-assets/blog/Show%20Us%20Your%20Calendar%20Images/Screen%20Shot%202021-09-13%20at%202.03.37%20PM.png)
+
+<p class="font-italic text-center">
+TJ coding live on Twitch.
+</p>
+
+Many would rightfully worry that lopping off one day a week wouldn’t sit well with a manager. TJ had the opposite experience: “No one cares about how you’re working as long as you’re getting done what you say you’re going to get done.” As long as his new schedule wasn’t a detriment to his team, TJ says, then he had his manager’s full support.
+
+TJ is frank: “I could work in the middle of the night for two nights a week and if I ship all the same amount of stuff, I don’t think people would care.” Though they might reach out about his health if it looked like he weren't sleeping, otherwise, his manager and his team just care about output.
+
+TJ recommends carrying over some software development principles over to your work calendar, namely: Don’t waterfall your calendar. “It’s unlikely that you’ll know the optimal schedule ahead of time,” TJ warns. “That’s not usually how we find the best version of something. We iterate on it over time.”
```

Comment:
Don't know if it makes much of a difference, but this sentence kinda comes out of nowhere. Maybe start it off with "When considering how to build your own work schedule, ..." or similar